storage/Utils/SystemCmd.h

```diff
@@ -203,6 +214,62 @@ namespace storage
     };
 
 
+    /**
+     * Exception class for SystemCmd. This is used both to really throw
+     * exceptions (if the 'DoThrow' behaviour was set for the SystemCmd) as
+     * well as for just logging the problem (done in both cases, 'DoThrow' and
+     * 'NoThrow'.
+     */
+    class SystemCmdException: public Exception
+    {
+    public:
+	SystemCmdException( SystemCmd * sysCmd, const string & msg ):
```

Comment:
Wouldn't two constructors be more appropriate that passing nullptr as the first argument?